### DISCUSSION TITLE
Is this normal, that my node Found 1 proof, but a havent got any XCH?

### DISCUSSION THREAD

**Discussion by norbul** [Chia-Network/chia-blockchain#4969] [Support]
Is this normal, that my node Found 1 proof, but a havent got any XCH?

`2021-05-13T01:33:57.685 harvester chia.harvester.harvester: INFO     3 plots were eligible for farming e266244858... Found 1 proofs. Time: 0.06891 s. Total plots`

**Comment by ilyakooo0** [comment]
Finding a proof does not guarantee winning. Many other farmers probably also found proofs for that block. The winner is the farmer who provided the "best" proof.

**Comment by OversizedWizard** [comment]
I've mined from when the network was only 200Pb. Had a week remaining it said. 
Still plotting and saying 2months left now 
Never received anything so far.

So far, total waste.

Started mining ETH using rtx3060 4 months ago, it's paid for itself.

This is not eco friendly as they think either.

Solo mining is now completely dead for small to average users.

Mining farms and pools only now it seems have some sort of chance.

Ah well, least someone will buy the hard drives I've wasted for double price in few weeks when shortage happens pmsl

**Reply by norbul** [reply]
Check your configuration and if port 8444 is open. (You can check using https://www.yougetsignal.com/tools/open-ports/)
Friend of mine use 300plots and have already 8XCH -  starting like you. 
I hope you will see your first XCH ;-)

**Reply by OversizedWizard** [reply]
Ports are all good.  Plots are passing filters 

Just no rewards yet. Don't hold my hopes high to be honest

**Reply by norbul** [reply]
Have you checked your log (INFO), and have something like this?
```
2021-05-14T18:17:15.266 full_node chia.full_node.full_node: INFO     ⏲️  Finished signage point 52/64:... 
2021-05-14T18:17:16.276 harvester chia.harvester.harvester: INFO     2 plots were eligible for farming ... Found 0 proofs. Time: 0.99712 s. Total plots
2021-05-14T18:17:24.159 full_node chia.full_node.full_node: INFO     🌱 Updated peak to height ..., weight ..., hh .., forked at ..., rh: ..., total iters: ..., overflow: False, deficit: 0, difficulty: 304, sub slot iters: ..., Generator size: 6382, Generator ref list size: 1
2021-05-14T18:17:25.629 full_node chia.full_node.full_node: INFO     🌱 Updated peak to height ..., weight ..., hh ..., forked at ..., rh: ..., total iters: ..., overflow: False, deficit: 0, difficulty: 304, sub slot iters: ..., Generator size: No tx, Generator ref list size: No tx
2021-05-14T18:17:35.923 harvester chia.harvester.harvester: INFO     4 plots were eligible for farming ... Found 0 proofs. Time: 0.75978 s. Total plots
```

**Reply by OversizedWizard** [reply]
Where's the log stored.  Chia puts itself across loads of folders lol

**Reply by norbul** [reply]
In chia config folder config/mainnet/log/debug.log
Probably in config.yaml you have to change log_level: WARNING to  log_level: INFO and restart chia node.

**Comment by OversizedWizard** [comment]
Thanks. 

Just let it run with changes and it's getting logged as farming. Shows similar to yours  1 plot eligible for farming, o proof found.

So it's doing the farming, just never got anything so far.

So far a total waste of time. 160 plots I have so far.  Now showing 2 months for reward .

Like I said when I first started it showed 1 week to reward 

At the rate it's growing, it'll never get a reward.

Talk about flogging a dead horse pmsl.

**Reply by norbul** [reply]
Check chia version should be at least 1.1.5 and in logs checking proof should be less than 5 seconds.

**Reply by padik11** [reply]
How can I check how long it takes to check the ID? Please explain in turn where I can find it.

**Reply by norbul** [reply]
padik11 - you have to change in config.yaml log_level from WARNING to INFO
Then in logs (.chia/mainnet/log/debug.log) you are looking for lines like this
`2021-05-01T01:07:51.126 harvester chia.harvester.harvester: INFO     1 plots were eligible for farming... Found 0 proofs. Time: 0.54682 s. Total XX plots`

And there is Time: how long takes checking proof

**Reply by padik11** [reply]
Thank you

**Reply by padik11** [reply]
How can I check it. Can you describe step by step?

**Reply by norbul** [reply]
Maybe this will help 
https://github.com/Chia-Network/chia-blockchain/wiki/How-to-Check-If-Everything-is-Working-(or-Not)

**Reply by padik11** [reply]
Thank you

**Reply by padik11** [reply]
I have a question. Everything works normally, there is synchronization, I connect to perr but I have logs in the log files:

2021-05-16T07: 45: 44.999 full_node chia.full_node.full_node: ERROR Consensus error 124, not disconnecting
2021-05-16T07: 47: 30.312 harvester chia.plotting.plot_tools: WARNING Directory: E: \ chia_dzialka_tymczasowyx does not exist.

Is something wrong? This is a temporary file.

**Reply by padik11** [reply]
Thank You.

**Comment by xorinox** [comment]
`2021-06-17T08:53:28.233 harvester chia.harvester.harvester: INFO     5 plots were eligible for farming 8e5dece611... Found 1 proofs. Time: 0.99216 s. Total 693 plots`   once in a while happens and no explanation can be found. That one is from today.

**Comment by BrandtH22** [comment]
@norbul , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
